src/calculateRentalCost.js

```diff
@@ -29,6 +29,25 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const threeDaysDiscount = 20;
+  const weekDiscount = 50;
+  const total = days * price;
+
+  switch (true){
+
+    case days > 6 :
+      return total - weekDiscount;
+
+    case days > 2 :
+      return total - threeDaysDiscount;
+
+    case days > 0 :
+      return total;
+
+    case true :
+      return 0;
```

Comment:
add `default` block to switch instead of `case true`
the `default` block  should always be in switch